cmake/oss.cmake.in

```diff
@@ -146,8 +146,8 @@ ExternalProject_Add(HdrHistogram
   DEPENDS ${default_depends})
 
 ExternalProject_Add(abseil
-  GIT_REPOSITORY https://github.com/vectorizedio/abseil-cpp.git
-  GIT_TAG 1e3d25b2657228bd691ee938cfd37d487f48054b
+  GIT_REPOSITORY https://github.com/abseil/abseil-cpp.git
+  GIT_TAG 20210324.1
```

Comment:
hmm... why not update our tree instead ?

Reply:
Please push our tree and I'll update the tag. We don't have any diffs against upstream, I was actually going to download the tarball and push to our cloud storage, but I forgot.